java/code/src/com/redhat/rhn/common/hibernate/HibernateFactory.java

```diff
@@ -63,7 +67,7 @@
 
     private static ConnectionManager connectionManager = new ConnectionManager();
     private static final Logger LOG = Logger.getLogger(HibernateFactory.class);
-    private static final int LIST_BATCH_MAX_SIZE = 1000;
+    public static final int LIST_BATCH_MAX_SIZE = 1000;
```

Comment:
Do you think, it would make sense to extract this value to a config file e.g in rhn.conf and use 1000 as default in case there is nothing defined?

Reply:
Question would be: is the user any better able than us developers to determine the right value? How would you inform a user about the choice?

Reply:
Yeah, that's a valid question. Ok let's leave it as it is then. I was of a view that in case the user has a huge setup and enough memory, maybe increasing batch size could work in a better performance.

Reply:
Hmm, makes sense. I'd go for the simplest solution now, we can always add a parameter as soon as we find a concrete case where this actually helps.